wallet-webapps/src/main/webapp/WEB-INF/jsp/walletBalance.jsp

```diff
@@ -11,9 +11,17 @@
 <%@ page import="org.exoplatform.wallet.model.ContractDetail"%>
 <%@ page import="org.exoplatform.wallet.model.Wallet"%>
 <%@ page import="org.exoplatform.wallet.model.WalletType"%>
+<%@ page import="org.exoplatform.portal.application.PortalRequestContext"%>
+<%@ page import="org.exoplatform.web.application.RequestContext"%>
+
+
 <%
   String title = "Wallet";
+  String walletUrl = "";
+
   try {
+    PortalRequestContext requestContext = ((PortalRequestContext) RequestContext.getCurrentInstance());
+    walletUrl = "/portal/" + requestContext.getPortalOwner() + "/wallet";
```

Comment:
Could you use `org.exoplatform.social.core.service.LinkProvider#getPortalName` to replace the string *portal* too